### PR TITLE
Improve banner CSS

### PR DESCRIPTION
**Pull request by jonasfj**
* On standalone pages the banner will sit in page and be hidden.
 * On package page (and landing page) the banner will float over content with position absolute, as these pages have sufficient margin to allow for this.

In both cases content will not jump when the banner is dismissed, or when the banner is loaded.

### CODE REVIEW

**Review by isoos — APPROVED** [review]
Can you push this to staging?

**Comment by jonasfj** [comment]
Pushed to `staging-b`

File: pkg/web_css/lib/src/_base.scss [hunk 1]
```diff
@@ -393,31 +393,20 @@ pre {
   text-align: center;
 
   .dismisser {
-    float: right;
+    position: absolute;
```
Comment (isoos) [inline pkg/web_css/lib/src/_base.scss:396]: Not sure: does this need a `position: relative` in the parent element?

**Review by isoos — APPROVED** [review]

File: pkg/web_css/lib/src/_base.scss (hunk 1 above)
Comment (jonasfj) [reply pkg/web_css/lib/src/_base.scss:396]: I don't think so...

**Pull request merged**